### PR TITLE
Update azuread.md

### PR DESCRIPTION
**Pull request by salihzett**
just the docu for clarifying the setting in Azure AD.
https://github.com/grafana/grafana/issues/23358

### CODE REVIEW

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Looks pretty good, but all UI elements that you click need to be bold, not italic. Please review the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md)

**Review by oddlittlebird — APPROVED** [review]

**Comment by oddlittlebird** [comment]
Sorry, I must have missed this the first time around. Thanks for contributing to the docs!

**Comment by oddlittlebird** [comment]
@aknuds1 , can you fix the build pipeline for this?

File: docs/sources/auth/azuread.md [hunk 1]
```diff
@@ -43,6 +43,11 @@ To enable the Azure AD OAuth2 you must register your application with Azure AD.
 
 1. Click **Add** then copy the key value, this will be the OAuth client secret.
 
+1. Click **Token configuration** and click *Add groups claim*
+    - Choose *Security groups* or other group types which you use in your AzureAD. If you are not sure, choose every group type.
+    - Customize token properties by type: For ID, Access and SAML click *Group ID*
+    - In Addtional: Click *Add optional claim*, choose Token type ID, Access and SAML and claim to *email*.
```
Comment (aknuds1) [inline docs/sources/auth/azuread.md:49]:
```suggestion
    - In Additional: Click *Add optional claim*, choose Token type ID, Access and SAML and claim to *email*.
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]
There's a typo, which breaks the build.

**Comment by aknuds1** [comment]
@oddlittlebird A typo in this PR broke the build, also the CLA isn't signed.

**Comment by daniellee** [comment]
@salihzett can you sign the CLA please so that we can merge this PR?

**Pull request closed without merging**